linode/accountavailabilities/datasource_test.go

```diff
@@ -0,0 +1,37 @@
+//go:build integration
+
+package accountavailabilities_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/linode/terraform-provider-linode/v2/linode/acceptance"
+	"github.com/linode/terraform-provider-linode/v2/linode/accountavailabilities/tmpl"
+)
+
+func TestAccDataSourceAccountAvailabilities_basic(t *testing.T) {
```

Comment:
I originally wanted to add a test case for filtering on the `unavailable` field, but as far as I could tell there isn't a reliable way to guarantee a service type would be unavailable in the production environment.

Reply:
Yes AFAIK it shouldn't be a service that unavailable to a customer permanently in a region 😃 And currently there isn't `unavailable` data backfilled.